core/src/replay_stage.rs

```diff
@@ -1682,6 +1682,21 @@ impl ReplayStage {
         blockstore
             .set_dead_slot(slot)
             .expect("Failed to mark slot as dead in blockstore");
+
+        let ((min_fec_set_count, last_fec_set_count), removed_stats) = {
+            let mut stats = blockstore.turbine_fec_set_stats.lock().unwrap();
+            let min_counts = stats.get_min_index_count(&slot);
+            let removed = stats.remove(&slot);
+            (min_counts, removed)
+        };
+        datapoint_info!(
+            "replay-stage-mark_dead_completed_unrepaired_slot",
+            ("slot", slot, i64),
+            ("min_turbine_fec_set_count", min_fec_set_count, i64),
+            ("last_turbine_fec_set_count", last_fec_set_count, i64),
+            ("turbine_only", removed_stats, bool),
+        );
```

Comment:
This thing can also be defined as a method on `TurbineFecSetStats`. Something like:
```rust
fn submit_slot_stats(name: &'static str, slot: Slot) { ... }
```
Though then you need to move `RwLock` inside the struct so that the submit-slot-stats function does not hold on to the lock for too long.